Command /ctgstats need to be fixed

MecLoving opened the pull request:
the command /ctgstats doesn't display the number or reinforcement in a group anymore. Instead, it displays the same information as /ctgroupinfo without the text format.

erocs committed:
Rail protection from water/lava
Inspired by https://github.com/remyroy/Citadel.git

erocs committed:
Remove development comment block

erocs committed:
Add Faction discipline flags

Commit:
Added permission nodes for ctgstats,ctpstats and the group node citadel.admin

erocs committed:
Citadel permission updates
- Add /ctdiscipline and /ctremediate
- Remove ConsoleCommand handler
- Expose admin commands both in-game and to the console
- Break the PlayerReinforcement->Faction dependency

erocs committed:
Fix NPE

erocs committed:
Add additional admin overrides

erocs committed:
Fix Citadel to work nicely with Essentials

fndragon committed:
Added blockhandler to disallow placing signs above chest blocks if the
user placing the sign is not allowed to access the chest below.  This
neatly solves the interaction with PhysicalShop.

fndragon committed:
Merged the listeners together so we only have one listener.  Also added
check for PhysicalShop

rourke750 committed:
Update BlockListener.java

fix for 1.5 by cancelling block breaks for placing blocks with hoppers.

rourke750 committed:
Update BlockListener.java

added stopping chest placement next to reinforced chests.

erocs committed:
Merge pull request #14 from fndragon/master

Update to eliminate chest thievery.

erocs committed:
Merge/refactor Rourke's cb73175291 and dfee4aec4a
+ Fix whitespace

erocs committed:
Activate 1.5 specifics from 1255b61b18

erocs committed:
Merge branch 'master' of https://github.com/erocs/Citadel

erocs committed:
Merge branch 'master' into 1.5

erocs committed:
Only cancel /cti block click in creative mode

erocs committed:
Fixup whitespace

erocs committed:
Always save default config.yml into plugins dir

erocs committed:
Make /ctgstats available to players

erocs committed:
Prevent object growth from subverting reinforcements

erocs committed:
Remove dup code and fix left-over copy.

erocs committed:
Determine securable by InventoryHolder superclass

erocs committed:
Support trapped chests

erocs committed:
Merge branch '1.5'

Conflicts:
	plugin.yml

erocs committed:
Rev version to keep unique version for merged 1.5

erocs committed:
Add CreateReinforcementEvent

erocs committed:
Fire CreateReinforcementEvent on /ctr owner change

erocs committed:
Prevent rails under reinforced containers
This is to prevent minecart hoppers from pulling items from reinforced
 containers.

erocs committed:
Fix HopperCarts, add plant reinforcements

erocs committed:
Add the idx_reinforcement_chunkid index

erocs committed:
Remove extraneous StructureGrowEvent

erocs committed:
Protect reinforced crops from water buckets

erocs committed:
Owners can break reinforced crops

erocs committed:
Add group caching

erocs committed:
Crop auto-bypass also for group members

erocs committed:
Merge branch 'master' into dev

Conflicts:
	plugin.yml

erocs committed:
Fix some group cache bugs

erocs committed:
Add GroupChangeEvent

erocs committed:
Fix another group cache bug
We might have to also add the version row to all group related tables to
make ebean happy.

erocs committed:
Fix GroupChangeEvent bug

erocs committed:
Fix hoppers with double chests

erocs committed:
Sugar cane grows on sand and grass

erocs committed:
Reinforcement maturation and scaling

erocs committed:
Add acid block, adjust /cti

erocs committed:
Add /ctmature

erocs committed:
Propagate maturation, prevent insta-acid

erocs committed:
Scaled materials shouldn't break at scale unless immature

erocs committed:
Async queries for stats commands

shadowjay1 committed:
Implement PlayerDamageReinforcementEvent

shadowjay1 committed:
Update version.

shadowjay1 committed:
Merge branch 'dev'

Conflicts:
	plugin.yml

erocs committed:
Merge pull request #46 from shadowjay1/master

Implement PlayerDamageReinforcementEvent.

erocs committed:
Fix acid block when maturity is disabled

erocs committed:
Merge branch 'dev' of github.com:erocs/Citadel into dev

Conflicts:
	plugin.yml

erocs committed:
Add license

erocs committed:
Make ACTIVATOR_RAIL same as other rails

erocs committed:
Don't display (Hardening) for /cti if not applicable

erocs committed:
Make PlRein message info which is governed by verbose logging setting

erocs committed:
Prevent reinforced wheat trampling

erocs committed:
Add acid block reinforcement tax

erocs committed:
Use PlayerInteract instead of BlockPhysics for preventing crop trample

erocs committed:
Rev version

erocs committed:
Unreinforced crops should break

erocs committed:
Async /ctstats

erocs committed:
Fix warnings

erocs committed:
Fix issue with reinforcement materials and lore

erocs committed:
Refactor InventoryMoveItemEvent, fix NPE, GRASS->LONG_GRASS

erocs committed:
Relax the hopper ownership check a bit

erocs committed:
Add support for insecure Hopper transfers

erocs committed:
Fix reinforced dirt->crop transformation

erocs committed:
Configurable verbose messages

erocs committed:
Update PlayerReinforcement to verify via Faction.isFounder

erocs committed:
Add DB versioning, update /ctdel for DB lag issue

erocs committed:
Improve acid block targeting

erocs committed:
Fix acid blocks w/ double chests, fix a couple format strings

rourke750 committed:
Merge pull request #1 from erocs/master

um please dont let this be a fuck up

rourke750 committed:
fixed bug where you could break invulnerable blocks

rourke750 committed:
Update plugin.yml

erocs committed:
Add to non-reinforceables

erocs committed:
Fix melon/pumpkin independent reinforcements

dquist committed:
Fix group deletion on plugin start

dquist committed:
Add batch updating of reinforcement records

Perform batch reinforcement updates for deleted groups and only deletes
groups if there are no more records to update for that group.

dquist committed:
Added config items
the delete flag is explicitly checked now

dquist committed:
Fixed comments

dquist committed:
Update some comments and formatting

dquist committed:
Update plugin.yml

Increment version number

dquist committed:
restore loadDeletedGroups

ttk2 committed:
Merge pull request #1 from erocs/upstream

Update to live version

WildWeazel committed:
updating build script for jenkins

ttk2 committed:
Merge pull request #2 from WildWeazel/patch-1

updating build script for jenkins

WildWeazel committed:
java target because maven is dense

grumble grumble

ttk2 committed:
Merge pull request #3 from WildWeazel/patch-1

java target because maven is dense

WildWeazel committed:
use bukkit repo and java 6

ttk2 committed:
Merge pull request #4 from WildWeazel/patch-1

use bukkit repo and java 6

ttk2 commented:
first off this is the wrong repo, we are using mine for the time
being, second, I know the admin version of this command still shows
number of items so chances are someone got confused with how the
command was supposed to work for some people vs admins.

Also does this check that you own the group before giving you the info?

On 7/18/14, MecLoving notifications@github.com wrote:

> the command /ctgstats doesn&#39;t display the number or reinforcement in a
> group anymore. Instead, it displays the same information as /ctgroupinfo
> without the text format.
> You can merge this Pull Request by running:
> 
>   git pull https://github.com/erocs/Citadel master
> 
> Or you can view, comment on it, or merge it online at:
> 
>   https://github.com/Exultant/Citadel/pull/105
> 
> -- Commit Summary --
> - Rail protection from water/lava
> - Remove development comment block
> - Add Faction discipline flags
> - Added permission nodes for ctgstats,ctpstats and the group node
>   citadel.admin
> - Citadel permission updates
> - Fix NPE
> - Add additional admin overrides
> - Fix Citadel to work nicely with Essentials
> - Added blockhandler to disallow placing signs above chest blocks if the
> - Merged the listeners together so we only have one listener.  Also added
> - Update BlockListener.java
> - Update BlockListener.java
> - Merge pull request #14 from fndragon/master
> - Merge/refactor Rourke&#39;s cb73175291 and dfee4aec4a
> - Activate 1.5 specifics from 1255b61b18
> - Merge branch &#39;master&#39; of https://github.com/erocs/Citadel
> - Merge branch &#39;master&#39; into 1.5
> - Only cancel /cti block click in creative mode
> - Fixup whitespace
> - Always save default config.yml into plugins dir
> - Make /ctgstats available to players
> - Prevent object growth from subverting reinforcements
> - Remove dup code and fix left-over copy.
> - Determine securable by InventoryHolder superclass
> - Support trapped chests
> - Merge branch &#39;1.5&#39;
> - Rev version to keep unique version for merged 1.5
> - Add CreateReinforcementEvent
> - Fire CreateReinforcementEvent on /ctr owner change
> - Prevent rails under reinforced containers
> - Fix HopperCarts, add plant reinforcements
> - Add the idx_reinforcement_chunkid index
> - Remove extraneous StructureGrowEvent
> - Protect reinforced crops from water buckets
> - Owners can break reinforced crops
> - Add group caching
> - Crop auto-bypass also for group members
> - Merge branch &#39;master&#39; into dev
> - Fix some group cache bugs
> - Add GroupChangeEvent
> - Fix another group cache bug
> - Fix GroupChangeEvent bug
> - Fix hoppers with double chests
> - Sugar cane grows on sand and grass
> - Reinforcement maturation and scaling
> - Add acid block, adjust /cti
> - Add /ctmature
> - Propagate maturation, prevent insta-acid
> - Scaled materials shouldn&#39;t break at scale unless immature
> - Async queries for stats commands
> - Implement PlayerDamageReinforcementEvent
> - Update version.
> - Merge branch &#39;dev&#39;
> - Merge pull request #46 from shadowjay1/master
> - Fix acid block when maturity is disabled
> - Merge branch &#39;dev&#39; of github.com:erocs/Citadel into dev
> - Add license
> - Make ACTIVATOR_RAIL same as other rails
> - Don&#39;t display (Hardening) for /cti if not applicable
> - Make PlRein message info which is governed by verbose logging setting
> - Prevent reinforced wheat trampling
> - Add acid block reinforcement tax
> - Use PlayerInteract instead of BlockPhysics for preventing crop trample
> - Rev version
> - Unreinforced crops should break
> - Async /ctstats
> - Fix warnings
> - Fix issue with reinforcement materials and lore
> - Refactor InventoryMoveItemEvent, fix NPE, GRASS-&gt;LONG_GRASS
> - Relax the hopper ownership check a bit
> - Add support for insecure Hopper transfers
> - Fix reinforced dirt-&gt;crop transformation
> - Configurable verbose messages
> - Update PlayerReinforcement to verify via Faction.isFounder
> - Add DB versioning, update /ctdel for DB lag issue
> - Improve acid block targeting
> - Fix acid blocks w/ double chests, fix a couple format strings
> - Merge pull request #1 from erocs/master
> - fixed bug where you could break invulnerable blocks
> - Update plugin.yml
> - Add to non-reinforceables
> - Fix melon/pumpkin independent reinforcements
> - Fix group deletion on plugin start
> - Add batch updating of reinforcement records
> - Added config items
> - Fixed comments
> - Update some comments and formatting
> - Update plugin.yml
> - restore loadDeletedGroups
> 
> -- File Changes --
> 
> ```
> D Citadel.jar (0)
> A LICENSE.txt (11)
> M bin/META-INF/MANIFEST.MF (4)
> M config.yml (24)
> M plugin.yml (52)
> M src/com/untamedears/citadel/Citadel.java (174)
> M src/com/untamedears/citadel/ConfigManager.java (104)
> M src/com/untamedears/citadel/GroupManager.java (95)
> M src/com/untamedears/citadel/GroupStorage.java (456)
> M src/com/untamedears/citadel/MemberManager.java (2)
> M src/com/untamedears/citadel/PlacementMode.java (6)
> M src/com/untamedears/citadel/Utility.java (385)
> M src/com/untamedears/citadel/access/AccessDelegate.java (50)
> M src/com/untamedears/citadel/access/BedAccessDelegate.java (6)
> M src/com/untamedears/citadel/access/ChestAccessDelegate.java (5)
> A src/com/untamedears/citadel/access/CropAccessDelegate.java (52)
> M src/com/untamedears/citadel/access/DoorAccessDelegate.java (7)
> M src/com/untamedears/citadel/command/CommandHandler.java (87)
> M src/com/untamedears/citadel/command/CommandUtils.java (17)
> D src/com/untamedears/citadel/command/ConsoleCommand.java (75)
> A src/com/untamedears/citadel/command/commands/AcidCommand.java (118)
> M src/com/untamedears/citadel/command/commands/AddModCommand.java (12)
> M src/com/untamedears/citadel/command/commands/AllowCommand.java (16)
> M src/com/untamedears/citadel/command/commands/ConsoleCommands.java
> ```
> 
> (66)
>     M src/com/untamedears/citadel/command/commands/CreateCommand.java (7)
>     M src/com/untamedears/citadel/command/commands/DeleteCommand.java (21)
>     M src/com/untamedears/citadel/command/commands/DisallowCommand.java
> (11)
>     A src/com/untamedears/citadel/command/commands/DisciplineCommand.java
> (65)
>     M src/com/untamedears/citadel/command/commands/FortifyCommand.java (4)
>     M src/com/untamedears/citadel/command/commands/GroupCommand.java (4)
>     M src/com/untamedears/citadel/command/commands/GroupInfoCommand.java
> (10)
>     M src/com/untamedears/citadel/command/commands/GroupStatsCommand.java
> (89)
>     M src/com/untamedears/citadel/command/commands/GroupsCommand.java (16)
>     A src/com/untamedears/citadel/command/commands/InsecureCommand.java
> (34)
>     M src/com/untamedears/citadel/command/commands/JoinCommand.java (11)
>     M src/com/untamedears/citadel/command/commands/LeaveCommand.java (12)
>     A src/com/untamedears/citadel/command/commands/MatureCommand.java (56)
>     M src/com/untamedears/citadel/command/commands/MembersCommand.java (4)
>     M src/com/untamedears/citadel/command/commands/ModeratorsCommand.java
> (4)
>     M src/com/untamedears/citadel/command/commands/PasswordCommand.java
> (11)
>     M src/com/untamedears/citadel/command/commands/PlayerStatsCommand.java
> (110)
>     M src/com/untamedears/citadel/command/commands/ReinforceCommand.java
> (4)
>     M src/com/untamedears/citadel/command/commands/RemoveModCommand.java
> (13)
>     A src/com/untamedears/citadel/command/commands/ReprieveCommand.java
> (49)
>     M src/com/untamedears/citadel/command/commands/StatsCommand.java (45)
>     M src/com/untamedears/citadel/command/commands/TransferCommand.java
> (24)
>     M src/com/untamedears/citadel/dao/CitadelCachingDao.java (28)
>     M src/com/untamedears/citadel/dao/CitadelDao.java (276)
>     A src/com/untamedears/citadel/entity/DbVersion.java (37)
>     M src/com/untamedears/citadel/entity/Faction.java (103)
>     A src/com/untamedears/citadel/entity/FactionDelete.java (38)
>     M src/com/untamedears/citadel/entity/FactionMember.java (15)
>     M src/com/untamedears/citadel/entity/IReinforcement.java (2)
>     M src/com/untamedears/citadel/entity/Moderator.java (15)
>     M src/com/untamedears/citadel/entity/NaturalReinforcement.java (8)
>     M src/com/untamedears/citadel/entity/PlayerReinforcement.java (132)
>     A src/com/untamedears/citadel/events/CreateReinforcementEvent.java (76)
>     A src/com/untamedears/citadel/events/GroupChangeEvent.java (64)
>     A src/com/untamedears/citadel/events/GroupChangeType.java (10)
>     A src/com/untamedears/citadel/events/PlayerDamageReinforcementEvent.java
> (57)
>     M src/com/untamedears/citadel/listener/BlockListener.java (367)
>     M src/com/untamedears/citadel/listener/EntityListener.java (18)
>     A src/com/untamedears/citadel/listener/InventoryListener.java (145)
>     M src/com/untamedears/citadel/listener/PlayerListener.java (270)
>     A src/com/untamedears/citadel/listener/WorldListener.java (26)
> 
> -- Patch Links --
> 
> https://github.com/Exultant/Citadel/pull/105.patch
> https://github.com/Exultant/Citadel/pull/105.diff
> 
> ---
> 
> Reply to this email directly or view it on GitHub:
> https://github.com/Exultant/Citadel/pull/105

Commit:
Disable /ctgstats

Commit:
Update plugin.yml

Commit:
Update pom.xml

Previous version was a bit out of date. Hopefully, this doesn't break anything...

erocs committed:
Merge pull request #71 from ttk2/master

Mainline POM changes for Jenkins

erocs committed:
Merge remote-tracking branch 'drjawa/master'

Conflicts:
	pom.xml
